Tests/Worker/Workers.cs

```diff
@@ -46,9 +48,14 @@ public ClassWorker(string taskType = "random_task_type")
             WorkerSettings = new WorkflowTaskExecutorConfiguration();
         }
 
-        public TaskResult Execute(Task task)
+        public async Task<TaskResult> Execute(Conductor.Client.Models.Task task, CancellationToken token)
         {
-            throw new System.Exception("random exception");
+            if (token.IsCancellationRequested)
+            {
+                throw new Exception("Token request Cancelled");
+            }
+
+            throw new Exception("random exception");
```

Comment:
What is random exception here? Why do we need to throw exception here? Can we make the message proper?